hail/python/hail/vds/combiner/combine.py

```diff
@@ -30,16 +30,16 @@ def make_variants_matrix_table(mt: MatrixTable,
 
     transform_row = _transform_variant_function_map.get((mt.row.dtype, info_key))
     if transform_row is None or not hl.current_backend()._is_registered_ir_function_name(transform_row._name):
-        def get_lgt(e, n_alleles, has_non_ref, row):
-            index = e.GT.unphased_diploid_gt_index()
+        def get_lgt(gt, n_alleles, has_non_ref, row):
+            index = gt.unphased_diploid_gt_index()
```

Comment:
Is this function valid on phased GTs? Do we need to unphase it first?

Reply:
we do thanks for checking